Upstream: Should be
Reason: Fixes compilation on big-endian arches like s390x

diff --git a/src/modules/loaders/loader_argb.c b/src/modules/loaders/loader_argb.c
index ec58ca3..e696fe4 100644
--- a/src/modules/loaders/loader_argb.c
+++ b/src/modules/loaders/loader_argb.c
@@ -52,7 +52,7 @@ load2(ImlibImage * im, int load_data)
            goto quit;
 
 #ifdef WORDS_BIGENDIAN
-        for (l = 0; l < im->w; l++)
+        for (int l = 0; l < im->w; l++)
            SWAP_LE_32_INPLACE(ptr[l]);
 #endif
         ptr += im->w;
